sdk/daml-assistant/src/DA/Daml/Assistant/Command.hs

```diff
@@ -120,7 +120,7 @@ installParser = InstallOptions
     <*> fmap BashCompletions (flagYesNoAuto' "bash-completions" "Install bash completions for Daml assistant. Default is yes for linux and mac, no for windows." idm)
     <*> fmap ZshCompletions (flagYesNoAuto' "zsh-completions" "Install Zsh completions for Daml assistant. Default is yes for linux and mac, no for windows." idm)
     <*> fmap InstallWithInternalVersion (flagYesNoAuto "install-with-internal-version" False "Allow installing from a tarball that has no associated release by using the tarball's SDK version as its release version." internal)
-    <*> fmap InstallWithCustomVersion (optionOnce (Just <$> str) (long "install-with-custom-version" <> value Nothing <> help "Install a tarball with a custom release version. Useful for environments without internet access, where Daml cannot look up the release version from Github."))
+    <*> fmap InstallWithCustomVersion (optionLast str (long "install-with-custom-version" <> help "Install a tarball with a custom release version. Useful for environments without internet access, where Daml cannot look up the release version from Github."))
```

Comment:
Since we already tell some of our users to call `install.sh` with `--install-with-custom-version=<version>`, 

> You can override the lookup to the internet via the `--install-with-custom-version <version>` optional argument. This is the current approach for environments that are internet-access-restricted.
> -- https://support.digitalasset.com/s/case/500Pq00000CJ2NQIA1/installation-of-sdk283-on-win10-system-failed 

and given that this PR embeds `--install-with-custom-version=x` in `install.sh`, I've modified the CLI parser to accept it multiple times while only keeping the last value, in the off chance that some customer might still pass the extra flag

fyi @dylant-da

Reply:
Oh nice, I was dreading having to do that kind of parsing in Bash - or in batch, for that matter.

Thank you!